src/widgets/modals/ByProgramModal.js

```diff
@@ -382,20 +254,45 @@ const localStyles = StyleSheet.create({
     alignSelf: 'center',
     marginTop: 60,
   },
-  textInput: {
-    justifyContent: 'space-between',
-    paddingLeft: 20,
-    maxHeight: 35,
-    minWidth: 500,
-    marginTop: '2%',
+  pageButtonTextStyle: {
+    ...globalStyles.buttonText,
+    color: 'white',
+    fontSize: 14,
   },
 });
 
+const getTitleString = type =>
+  `${type === 'stocktake' ? programStrings.stock_take : navStrings.requisition} ${
```

Comment:
I'm a fan of one-liners, but think could probably use a variable here to break it up and make it a bit easier to follow. Something like:

```
const getTitleString = type => {
  const { stock_take: stocktakeString, details: detailsString } = programStrings;
  const { requisition: requisitionString } = navStrings;
  const typeString = type === 'stocktake' ? stocktakeString : requisitionString
  return `${typeString} ${detailsString}`
}
```